### ISSUE TITLE
Add version to download URL

### ISSUE THREAD

**Issue by QuLogic** [nvkelso/natural-earth-vector#246]
It would be nice if the downloads all contained some version specifier, either in the path or the filename. The current unversioned link could redirect to the latest version.

For reproducibility's sake, I'd like to be able to point to a versioned URL that _never_ changes. I can understand if you don't want to continuously host the old versions, but a broken link is better than one that changes without warning in this case.

**Comment by noirbizarre** [comment]
Same for me. Our pipeline is broken because file has been replaced with the same URL and metadata format changed (from lower to upper case) and some metadata disappeared (lots of countries lost their `ISO_A2` and `ISO_3`).

Having versionned URL can prevent this issue.

**Comment by QuLogic** [comment]
Based on #445, it appears we can get a versioned link from `https://naciscdn.org/naturalearth/${version}/${scale}/${theme}/${scale}_${theme}.zip`.

But unfortunately, while #581 says to use https://naturalearth.s3.amazonaws.com/, the listing there does _not_ appear to have versioned files.

**Comment by nvkelso** [comment]
Yes, both endpoints offer versioned files, including the S3 bucket:

- https://naturalearth.s3.amazonaws.com/naturalearth/${version}/${scale}_${theme_group}/ne_${scale}_${theme}.zip

_NOTE: S3 is different than the NACIS CDN by grouping `${scale}_${theme_group}` in a single directory name versus nested child directory for the theme_group. This more closely matches files on Github, while NACIS CDN uses the legacy structure from the naturalearthdata.com website._

Reformatting your example:

- https://naciscdn.org/naturalearth/${version}/${scale}/${theme_group}/ne_${scale}_${theme}.zip


Where: 

- `${version}` is replaced with 5.1.2 or similar. 
    - This is fully qualified for **major.minor.patch** (not offered for **major** alone or **major.minor** alone)
- `${scale}` - where **10m**, **50m**, or **110m**
- `${theme_group}` - where **cultural**, **physical**, or **raster**
- `${theme}` - **name of theme** (without ne prefix or scale)

**Comment by QuLogic** [comment]
Ah, I see S3 does have the versions now, but it doesn't have 5.1.* yet.
